sdk/azcore/arm/runtime/poller.go

```diff
@@ -4,6 +4,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+// Deprecated: use azcore/runtime instead.
```

Comment:
Does this put a deprecation notice on the entire package?

Reply:
No, in fact it has no discernible effect, other than on the reader if they happen to see this.  I felt like I should add _something_, but perhaps the entry in the changelog is enough.

I don't feel too strong about this, so I can remove it if there are any concerns.

Reply:
I was curious enough to try this out. godoc apparently uses the first line of comment it finds on a package declaration as the synopsis of that package:
![image](https://user-images.githubusercontent.com/10964656/164787122-eca22092-6da9-4aca-bd5b-bfe895a382ab.png)
But this probably has no impact because there's no semantic effect and the audience is probably only SDK devs.

Reply:
Thanks for checking the docs.  I've updated the deprecation notice.